src/libraries/System.Transactions.Local/src/System.Transactions.Local.csproj

```diff
@@ -40,13 +42,81 @@
+    <Compile Include="System\Transactions\DtcProxyShim\DtcInterfaces\ITransactionTransmitterFactory.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\DtcInterfaces\ITransactionVoterBallotAsync2.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\DtcInterfaces\ITransactionVoterFactory2.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\DtcInterfaces\ITransactionVoterNotifyAsync2.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\DtcProxyShimFactory.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\EnlistmentNotifyShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\EnlistmentShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\Guids.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\NativeEnums.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\NotificationShimBase.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\OletxHelper.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\OletxXactTransInfo.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\Phase0NotifyShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\Phase0Shim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\ResourceManagerNotifyShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\ResourceManagerShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\TransactionNotifyShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\TransactionOutcome.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\TransactionShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\VoterNotifyShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\VoterShim.cs" />
+    <Compile Include="System\Transactions\DtcProxyShim\Xactopt.cs" />
+    <Compile Include="System\Transactions\Oletx\DtcTransactionManager.cs" />
+    <Compile Include="System\Transactions\Oletx\OletxCommittableTransaction.cs" />
+    <Compile Include="System\Transactions\Oletx\OletxDependentTransaction.cs" />
+    <Compile Include="System\Transactions\Oletx\OletxEnlistment.cs" />
+    <Compile Include="System\Transactions\Oletx\OletxResourceManager.cs" />
+    <Compile Include="System\Transactions\Oletx\OletxTransaction.cs" />
+    <Compile Include="System\Transactions\Oletx\OletxTransactionManager.cs" />
+    <Compile Include="System\Transactions\Oletx\OletxVolatileEnlistment.cs" />
+    <Compile Include="System\Transactions\TransactionInterop.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetPlatformIdentifier)' != 'windows'">
+    <Compile Include="System\Transactions\NonWindowsUnsupported.cs" />
+  </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\System.Diagnostics.EventLog\src\System.Diagnostics.EventLog.csproj" />
+    <ProjectReference Include="..\..\System.Configuration.ConfigurationManager\src\System.Configuration.ConfigurationManager.csproj" />
```

Comment:
These references are bad. 

`System.Transactions.Local` is a inbox assembly that ships as part of the .NET runtime.

System.Diagnostics.Eventing and System.Configuration.ConfigurationManager are standalone nuget packages.

These references means that you have to manually install these two packages for `System.Transactions.Local` to work. It is a regression and a bad experience.

Are these dependencies really required to make this work? Can you get rid of them?

These dependencies are also the reason for the mysterious build failures that you are seeing since they cause the referenced projects to build in new configurations that they are not meant to build for.

Reply:
Thanks @jkotas, makes sense! These were needed at some phase of the port from netfx but no longer appear to be, am removing.